lockbox-ios/View/ItemListView.swift

```diff
@@ -268,32 +268,7 @@ extension ItemListView {
     }
 
     private var sortingButton: UIButton {
-        let button = UIButton()
-        button.adjustsImageWhenHighlighted = false
-
-        let sortingImage = UIImage(named: "down-caret")?.withRenderingMode(.alwaysTemplate)
-        button.setImage(sortingImage, for: .normal)
-        button.setTitle(Constant.string.aToZ, for: .normal)
-
-        button.contentHorizontalAlignment = .left
-        button.setContentCompressionResistancePriority(.defaultLow, for: .horizontal)
-        button.titleEdgeInsets = UIEdgeInsets(top: 0, left: 5, bottom: 0, right: -20)
-        button.setTitleColor(.white, for: .normal)
-        button.setTitleColor(UIColor(white: 1.0, alpha: 0.6), for: .highlighted)
-        button.setTitleColor(UIColor(white: 1.0, alpha: 0.6), for: .selected)
-        button.setTitleColor(UIColor(white: 1.0, alpha: 0.6), for: .disabled)
-        button.tintColor = .white
-        button.translatesAutoresizingMaskIntoConstraints = false
-
-        button.addConstraint(NSLayoutConstraint(
-                item: button,
-                attribute: .width,
-                relatedBy: .equal,
-                toItem: nil,
-                attribute: .notAnAttribute,
-                multiplier: 1.0,
-                constant: 100)
-        )
+        let button = UIButton(title: Constant.string.aToZ, imageName: "down-caret")
```

Comment:
This button looks funny with these changes -- a little too spaced out on `A-Z` and truncates `Recent` when selecting the other sorting option.